README.md

```diff
@@ -1,44 +1,25 @@
 # VA.gov Team Repository
-*Last updated: [9/17/2019](#document-history)*
 
 This project aims to improve the lives of Veterans, their families, and their caregivers by facilitating the creation of interactive experiences on VA.gov on par with the private sector. It's comprised of a platform (the Veteran-facing Services Platform, aka VSP) which serves as the common framework for building cohesive products on VA.gov, and the VA.gov products themselves (the Veteran Facing Services, aka VFSs).
 
-This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs.
+This repository is used to store *non-code* documents and files created and used by teams building the VSP and VFSs. It is also used for issue tracking for many parts of the VSP platform.
 
 ***An important note about security for contributors to this repository: this is a public, open source space! Which means it's great for collaborating, but decidedly not great for storing sensitive information (like credentials, PII, etc). So be mindful as you work in this repo. Don't store anything sensitive here!***
 
 For information on how to use this repo, see the [Repo Guidelines](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/repo-guidelines.md).
 
-#### Top level stats:
```

Comment:
These stats were added by @rroueche in September 2019, I believe, so I would want to check with her on ownership of these and whether we should delete or assign someone to keep them updated. I can ask in Slack.

Reply:
IMO it's a bad idea to have such things committed to a readme. They will always be out of date. Instead we could provide a link to https://www.va.gov/performance-dashboard or the google analytics dashboard for those with access?

Reply:
Checked with Rachael — let's cut it for now, and can add back analytics later if/when we do a more complete overhaul of an intro page of sorts.